e2e/cypress/integration/team_settings/teams_spec.js

```diff
@@ -245,14 +245,14 @@ describe('Teams Suite', () => {
         cy.get('#teamDescription').should('be.visible').clear().type(teamDescription);
         cy.get('#teamDescription').should('have.value', teamDescription);
 
-        // Save new team description
-        cy.findByText(/save/i).click();
+        // Save and close
+        cy.uiSaveAndClose();
 
-        // # Close the team settings
-        cy.get('body').type('{esc}', {force: true});
+        // # Open team menu and click "Team Settings"
+        cy.uiOpenTeamMenu('Team Settings');
 
-        // Team tooltip should show description
-        verifyToolTip(teamDescription);
```

Comment:
Intentionally deleted as resolved by [MM-38386(https://mattermost.atlassian.net/browse/MM-38386). Tooltip is now no longer applied to team header button.
Test case MM-T2317 has been updated to reflect the change.